### DISCUSSION TITLE
Are Public hosted Vaultwarden instances legit .?

### DISCUSSION THREAD

**Discussion by Marsupoil76** [dani-garcia/vaultwarden#4367] [General]
I wanted to know if this site is legitimate for you, because personally I have a doubt... the domain name has lots of servers everywhere... and as Vaultwarden is not a company... which is hiding behind this supposed service

**Comment by BlackDex** [comment]
This project does not and will not ever host a publicly available instance.

Any publicly available Vaultwarden has no connection to this project except for it using this project provide server.

We do not know if those servers have modified the front or backend in any way. So we can't for sure say if they are safe or not.
